### PR TITLE
Multi-targeting exploration 1

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

**Comment by jaredpar** [comment]
retest this please

**Comment by dpoeschl** [comment]
These failures seem legitimate. @jaredpar please let me know if you feel they are unrelated.

**Comment by jaredpar** [comment]
@dpoeschl oh they're real and my fault. I have this PR marked as "Personal Review Only" because I know that it's not ready yet. Using Jenkins here to help show me everything I got wrong 😄

File: build/Targets/Settings.props [hunk 1]
```diff
@@ -101,6 +102,12 @@
   <!-- Unix specific settings -->
   <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
     <RoslynDebugType>portable</RoslynDebugType>
+
+    <!-- Presently can't build anything with a net46 TargetFramework on non-Windows. Disabling
+         for now. 
+        
+         https://github.com/dotnet/roslyn/issues/20932 -->
```
Comment (tannergooding) [inline build/Targets/Settings.props]: I linked the SDK issue tracking this (https://github.com/dotnet/sdk/issues/826) in your bug.

File: src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj [hunk 2]
```diff
@@ -9,8 +9,9 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.CSharp</RootNamespace>
     <AssemblyName>Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests</AssemblyName>
-    <TargetFramework>netstandard1.3</TargetFramework>
-    <RoslynProjectType>UnitTestPortable</RoslynProjectType>
+    <RoslynProjectType>UnitTest</RoslynProjectType>
+    <PackageTargetFallback>portable-net45</PackageTargetFallback>
```
Comment (tannergooding) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: I thought we got rid of all the dependencies that required this?

Comment (jaredpar) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: Yeah ... nope. xunit stuff sill lingers.

File: build/Targets/Settings.props (hunk 1 above)
Comment (jaredpar) [inline build/Targets/Settings.props]: Thanks!

File: src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj (hunk 2 above)
Comment (tannergooding) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: Ah, I hadn't realized we were still on an old enough version of xUnit for that to matter

Comment (jaredpar) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: It's amazing what you find out when you change things 😄

**Pull request closed without merging**